### DISCUSSION TITLE
Spinning at kol con clan house?

### DISCUSSION THREAD

**Discussion by xKiv** [loathers/garbage-collector#2021] [Q&A] [answered]
Is it normal for garbo to spend hours and hours doing nothing in Wanderers/Shocking Lick, just try to visit (in my case) kol con clan house (which does nothing because it is not available)?
No visible output for almost a minute, then the visit, then several switches of source terminal.
Over and over again.
Freshly (about 8 hours ago when I started) built mafia, freshly "git update"d garbo.
Today it happened after spending ~90 adventures, and there's still over 400 adventures left to spend.

If not, what should I tweak to avoid this?

**Comment by ajcoppa** [comment, marked as answer]
Your mafia still thinks the time-twitching tower is available. Visiting Seaside Town will refresh the `timeTowerAvailable` preference and stop garbo trying to visit that location.

**Reply by xKiv** [reply]
Thanks, that was (probably) all there was to it. I forgot the TTT is handled that way.
